packages/jira-adapter/test/adapter.test.ts

```diff
@@ -53,10 +58,63 @@ describe('adapter', () => {
     })
   })
   describe('deploy', () => {
-    it('should throw not implemented', async () => {
-      await expect(
-        adapter.deploy({ changeGroup: { groupID: '', changes: [] } })
-      ).rejects.toThrow(new Error('Not implemented.'))
+    let deployChangeMock: jest.Mock
```

Comment:
please use a proper type
```suggestion
    let deployChangeMock: jest.MockedFunction<typeof deployment.deployChange>
```
also, why do you need this is context to all tests?
seems like it is only used in the `beforeEach`, in can be defined internally there